build/configs/artik053/tc/defconfig

```diff
@@ -156,6 +157,12 @@ CONFIG_S5J_WATCHDOG=y
 CONFIG_S5J_WDT_DEFTIMEOUT=7000
 CONFIG_S5J_SFLASH=y
 # CONFIG_S5J_SENSOR_PPD42NS is not set
+CONFIG_S5J_PM=y
```

Comment:
ARTIK does not support pm on hw and architecture side. I don't agree enable pm on all of defconfig.

Reply:
yes, I have added to defconfigs which already support PM except tc/defconfig
I will remove it from artik053/tc defconfig.